pre-registration/pre-registration-core/src/main/java/io/mosip/analytics/event/anonymous/service/AnonymousProfileService.java

```diff
@@ -31,6 +31,9 @@ public class AnonymousProfileService implements AnonymousProfileServiceIntf {
 	@Value("${mosip.utc-datetime-pattern}")
 	private String utcDateTimePattern;
 
+	@Value("${mosip.preregistration.user.admin}")
+    private String userName;
```

Comment:
Can we rename this both as mosip.preregistration.anonymous-profile-username and anonymousProfileUsername ? Same in the test files

Reply:
Updated naming convention